tests/unit/unit-util.cpp

```diff
@@ -165,4 +237,8 @@ INSTANTIATE_TEST_SUITE_P(ParameterizedUnitUtilIntTests,
                          UnitUtilIntTests,
                          testing::ValuesIn(filter_solver_configurations({ TERMITER, THEORY_INT })));
 
+INSTANTIATE_TEST_SUITE_P(ParameterizedUnitUtilDimacsTests,
+                         UnitUtilDimacsTests,
+                         testing::ValuesIn(available_solver_configurations()));
```

Comment:
better to use `available_solver_enums` -- currently u r running each test twice.

Reply:
Yes, just realized now on seeing the code that both true and false are pushed for each enum.